base/shell/explorer/taskswnd.cpp

```diff
@@ -1386,20 +1385,11 @@ class CTaskSwitchWnd :
     {
         /* Only show windows that still exist and are visible and none of explorer's
         special windows (such as the desktop or the tray window) */
```

Comment:
You may move part (or all) of this comment inside the IsTaskWnd function. (Explaining what a "task wnd" is, somehow.)

Reply:
Moved.